chainerx_cc/chainerx/routines/indexing.cc

```diff
@@ -142,7 +142,6 @@ Array AddAt(const Array& a, const Array& indices, int8_t axis, const Array& b, I
     CHAINERX_ASSERT(0 <= axis && axis < a.ndim());
     CHAINERX_ASSERT(b.ndim() == indices.ndim() + a.ndim() - 1);
     CheckEqual(a.dtype(), b.dtype());
-
```

Comment:
Ditto.
By the way, as `AddAt` is now public, asserts should be exceptions instead (not to be done in this PR though).